### PR TITLE
Added error messages from Validator

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by vinicius73** [comment]
Nice! :+1:

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ composer.lock
 .DS_Store
 Thumbs.db
 .idea
+nbproject
```
Comment (tshafer) [inline .gitignore]: Remove

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
     {
       "name": "Taylor Otwell",
       "email": "taylorotwell@gmail.com"
+    },
+    {
+      "name": "Andrey Helldar",
+      "email": "helldar@ai-rus.com"
```
Comment (tshafer) [inline composer.json]: Remove

**Comment by tshafer** [comment]
Please add some tests.

**Comment by tshafer** [comment]
Please squash and fix merge conflicts.

**Comment by andrey-helldar** [comment]
How to write the tests? I can not.

**Comment by andrey-helldar** [comment]
I found the instructions.I read and add tests.

**Pull request closed without merging**